ankihub/db/db.py

```diff
@@ -638,9 +643,15 @@ def note_type_field_names(
         """Returns the names of the fields of the note type."""
         result = [
             field["name"]
-            for field in self.note_type_dict(
-                ankihub_did=ankihub_did, note_type_id=anki_note_type_id
-            )["flds"]
+            for field in sorted(
+                (
+                    field
+                    for field in self.note_type_dict(
+                        ankihub_did=ankihub_did, note_type_id=anki_note_type_id
+                    )["flds"]
+                ),
+                key=lambda f: f["ord"],
+            )
```

Comment:
:question: Is this change necessary? The fields should already be ordered. We were relying on this in `_AnkiHubDB.note_data`: https://github.com/AnkiHubSoftware/ankihub_addon/blob/df21cbbf71bccc6a3c6ba14dcb3c4acd1b3b666c/ankihub/db/db.py#L304

Reply:
Hmm doesn't seem to be strictly necessary but I just wanted to keep the explicit sorting done above in upsert_notes_data() and thought it would be better to add it here instead.